docs/dictionary/function/asin.lcdoc

```diff
@@ -35,19 +35,18 @@ The <asin> <function> <return|returns> a number between -pi/2 and pi/2.
 The result:
 The result of the <asin> <function> is <return|returned> in
 <radian|radians>. To get this result in <degree|degrees>, use the
-following <custom function>:.
+following <custom function>:
+
+    function asinInDegrees theMagnitude
+    return asin(theMagnitude) * 180 / pi
+    end asinInDegrees
```

Comment:
Could you change the `theMagnitude` parameter to `pMagnitude` please, and add indent to the `return` line of the code?